### ISSUE TITLE
Add label property

### ISSUE THREAD

**Issue by chanan** [chanan/BlazorStyled#4]
Emotion has the concept of label, which is basically a string that a user can append to the css class to help debug.

**Comment by chanan** [comment]
Sadly, this is not straight forward. The current IStyled looks like this:

```
Task<string> Css(string css);
Task<string> Css(string className, string css);
```

Which means I cant simply add:

```
Task<string> Css(string css, string label=null);
```

This issue will either require a breaking change or something like:

```
Task<string> CssWithLabel(string css, string label);
```

**Comment by chanan** [comment]
I decided to go with the option of:

```
Task<string> CssWithLabel(string css, string label);
```

**Comment by chanan** [comment]
Added the ability to append a name to the class name by sending a label property in the css:

```
div1 = await Styled.Css(@"
   color: hotpink;
   label: some-name;
");
```

Released in 1.0.0-preview6-05
